python/t/etc/Simple_SciPass_balancer_only.xml

```diff
@@ -3,6 +3,7 @@
    <switch dpid="0000000000000001" name="IUPUI-SW">
     <domain name="IUPUI" mode="SimpleBalancer" admin_status="active" least_specific_prefix_len="16" most_specific_prefix_len="24" 
 	    ignore_sensor_load="true" blacklist_priority="1000" whitelist_priority="900" sensor_min_load_threshold="0.1" 
+	    ipv6least_specific_prefix_len="64" ipv6most_specific_prefix_len="128" sensor_configurable_threshold="100"
```

Comment:
these values for IPv6 seem wrong... I would think the most specific would be /64 (there are only a few quadrillion of them... I guess I'm not sure how this actually could work seems like if you included anything you would have essentially infinite addresses?